### PR TITLE
[voice] Add voice commands for register/unregister dialogs and list them

### PR DESCRIPTION
**Pull request by GiviMAD**
This PR add new cli voice commands for exposing the new register/unregister dialog capability of the voice manager, plus commands for list existing dialog registrations and running dialogs.

Also fixes a command description issue where the parameter "hlis" was documented as "interpreters". I think that "hlis" is better as it is sorter, but let me know if there is a preference over "interpreters".

Related to https://github.com/openhab/openhab-core/issues/3265

### CODE REVIEW

File: bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java [hunk 1]
```diff
@@ -87,13 +93,21 @@ public List<String> getUsages() {
         return List.of(buildCommandUsage(SUBCMD_SAY + " <text>", "speaks a text"),
                 buildCommandUsage(SUBCMD_INTERPRET + " <command>", "interprets a human language command"),
                 buildCommandUsage(SUBCMD_VOICES, "lists available voices of the TTS services"),
+                buildCommandUsage(SUBCMD_DIALOGS, "lists the running dialog an its audio/voice services"),
```
Comment (lolodomo) [inline bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java:96]: typo: running dialog**s** an**d** **their** ...

File: bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java [hunk 2]
```diff
@@ -87,13 +93,21 @@ public List<String> getUsages() {
...
+                buildCommandUsage(SUBCMD_DIALOG_REGS,
+                        "lists the existing dialog registrations and its selected audio/voice services"),
```
Comment (lolodomo) [inline bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceConsoleCommandExtension.java:98]: and **their** ?

**Review by lolodomo — CHANGES_REQUESTED** [review]
Code looks good to me except one or two language typo.

**Comment by GiviMAD** [comment]
Thank you for the review. Should I update the documentation for these commands?

**Comment by lolodomo** [comment]
> Should I update the documentation for these commands?

Yes please, but wait for the merge first.

**Comment by GiviMAD** [comment]
Hey @openhab/core-maintainers, I saw yesterday the oh release schedule, do you think this change can be merged before the feature freeze?
It allows more visibility over the running dialogs and expose the register/unregister methods added for the v4.
Thank you.

**Review by J-N-K — APPROVED** [review]

**Review by kaikreuzer — APPROVED** [review]
Sure thing! Sorry for the delay.

**Pull request merged**